Jenkinsfile

```diff
@@ -123,6 +123,7 @@ pipeline {
         VERSION = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}"
         CLUSTER_NAME = sh(script: "echo jenkins-pxc-${GIT_SHORT_COMMIT} | tr '[:upper:]' '[:lower:]'", , returnStdout: true).trim()
         AUTHOR_NAME  = sh(script: "echo ${CHANGE_AUTHOR_EMAIL} | awk -F'@' '{print \$1}'", , returnStdout: true).trim()
+        IMAGE_HAPROXY="perconalab/percona-xtradb-cluster-operator:k8spxc-552-haproxy"
```

Comment:
please revert before merge

Reply:
https://github.com/percona/percona-docker/pull/378 is merged, please remove the line